packages/council-cli/.prettierrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  ...require("@council/prettier-config"),
+  plugins: [require("prettier-plugin-organize-imports")],
```

Comment:
not for this pr necessarily, but adding this to the @council/prettier-config would be nice i think

Reply:
Greed, will do in next PR